### ISSUE TITLE
IntegrityError: update or delete on table "api_machineimage" violates foreign key constraint. 

### ISSUE THREAD

**Issue by paraggit** [cloudigrade/cloudigrade#819]
## Summary
While running image inspection regression test cases on stage environment  'IntegrityError' seen.
Following is the detail description of the error message.
```
ForeignKeyViolation
update or delete on table "api_machineimage" violates foreign key constraint "api_machineimageinsp_machineimage_id_69ba8a76_fk_api_machi" on table "api_machineimageinspectionstart"
DETAIL:  Key (id)=(1367) is still referenced from table "api_machineimageinspectionstart".
```
Sentry Link.
- [IntegrityError](https://sentry.io/organizations/cloudigrade/issues/2236330246/?project=1270362&referrer=slack)

## Steps to Reproduce

This issue seen while running regression tests cases of images inspection.

## Expected Result

- There should not be any error get raised while running regression test suit.

## Actual Result

- Error ('IntegrityError' ) raised while running regression test suit.

**Comment by infinitewarp** [comment]
We need more precise and detailed info than simply "seen while running regression tests cases of images inspection" before we can work on this issue.

What exactly is the test suite doing when this error occurs? What API call is it executing? What were the expected and observed states of the system before and after that call? Does the error occur consistently and predictably when the test suite is performing that operation? Are there any relevant messages around the time of the error either from Kibana or in responses to the test suite's requests?

**Comment by paraggit** [comment]
I am trying to reproduce this issue but since sources is down today i couldn't able to made any progress.

**Comment by katherine-black** [comment]
@paraggit is there any update on reproduction steps for this issue?

**Comment by paraggit** [comment]
@katherine-black I was trying to reproduce this issue but no luck. I will move this issue to closed state for now and if it appear in future again i will open new issue for that with proper details.
